nym-vpn-core/crates/nym-vpn-lib/src/tunnel_state_machine/resolver.rs

```diff
@@ -360,6 +360,7 @@ impl LocalResolver {
 
 /// Flush the DNS cache.
 fn flush_system_cache() {
+    #[cfg(target_os = "macos")]
```

Comment:
The entire mod is excluded from all platforms except macos:

```rs
#[cfg(target_os = "macos")]
mod resolver;
```

Reply:
Yep, some remains from before